### PR TITLE
Libraries

### PR DESCRIPTION
**Pull request by withlovedese**
Added minimum viable product chart with hardcoded data.

### CODE REVIEW

**Review by ReshmaMary-gcloud — COMMENTED** [review]
Could you merge the branches so that only relevant file shows up in the pull request. Also make sure you commit the changes to add a specific functionality together, like servlet changes, pom.xml etc

**Comment by withlovedese** [comment]
Done

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -36,6 +36,34 @@ async function getCommentUsingAsyncAwait(){
     commentContainer.innerText = comments;
 }
 
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
```
Comment (ReshmaMary-gcloud) [inline portfolio/src/main/webapp/script.js:43]: Did you have to make any other changes than the script changes? a new pom dependency or script imports etc?

**Review by ReshmaMary-gcloud — COMMENTED** [review]
Hi Dese, this branch still shows the changes you had on the previous pull request. When you created the pull request did you base it out of master or your un merged previous branch? In order to show only the relevant changes you might need to base it out of the previous branch than master. Other option is to merge the approved branch to master before initiating the pull request. The sps program write up will give you a good idea.

**Pull request merged**